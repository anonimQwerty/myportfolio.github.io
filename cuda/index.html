<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<meta name="viewport" content="width=device-width, initial-scale=1.0">
	 <link rel="preconnect" href="https://fonts.googleapis.com">
	<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
	<link href="https://fonts.googleapis.com/css2?family=Titillium+Web:ital,wght@0,200;0,300;0,400;0,600;0,700;0,900;1,200;1,300;1,400;1,600;1,700&display=swap" rel="stylesheet"> 
	<link rel="stylesheet" href="css/style.css">
	
	
	<link rel="stylesheet" href="css/main.css">
	<title>Document</title>
</head>
<body>
<div class="firstblock">
	<div class="container">

		<header class="header">
			<div class="leftside">
				<img src="img/logo.png" alt="logo">
				</div>
				<div class="rightside">
					
						<a href="#home" class="header__link">home</a>
						<a href="#about" class="header__link">about</a>
						<a href="#work" class="header__link">work</a>
						<a href="#blog" class="header__link">blog</a>
						<a href="#contact" class="header__link">contact</a>
					
				</div>
			
		</header>
		<div class="firstblock__text">
		<p>Hi there! We are the new kids on the block and we build awesome websites and mobile apps.</p></div>

		<div class="firstblock__btn">
		<button class="">WORK WITH US!</button></div>
		




	</div>
	</div>

	<div class="servicesprovise">
		<div class="container">
			<h1 class="servise__title">SERVICES WE PROVIDE</h1>
			<p class="servise__subtitle">We are working with both individuals and businesses from all over the globe to create awesome websites and applications.</p>
			<div class="services__features">
				<div class="servise__feature">
					<img src="img/flag.png" alt="">
					<h3>Branding</h3>
					<p>Lorem ipsum dolor, sit amet consectetur adipisicing elit. Quos consequuntur, voluptatem sunt?</p>
				</div>
				<div class="servise__feature">
					<img src="img/pensil.png" alt="">
					<h3>Design</h3>
					<p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Possimus, laudantium sequi temporibus.</p>
				</div>
				<div class="servise__feature">
					<img src="img/gears.png" alt="">
					<h3>Development</h3>
					<p>Lorem, ipsum dolor sit amet consectetur adipisicing, elit. Ratione reprehenderit rem perferendis.</p>
				</div>
				<div class="servise__feature ">
					<img src="img/rocket.png" alt="">
					<h3 class="servise__sucking_block">ROCKET SCIENCE</h3>
					<p>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Ad explicabo iure sit?</p>
				</div>

			</div>
		</div>
	</div>

	<div class="ourteam">
		<div class="container">
			<h2 class="ourteam_team">MEET OUR BEAUTIFUL TEAM</h2>
			<p class="ourteam_subtitle">We are a small team of designers and developers, who help brands with big ideas.</p>
			<div class="ourteam_peoples">
				<div class="ourteam_people">
					<img src="img/people.jpg" alt="" class="people_photo">
					<h4 class="name">ANNE HATHAWAY</h4>
					<p class="position">CEO / Marketing Guru</p>
					<p class="description">Nemo enim ipsam voluptas sit aspernatur aut odit aut fugit, sed quia consequuntur magni dolores eos qui ratione voluptatem nesciunt.</p>
					<div class="people_sotialnetworks"><p class="icon-facebook"></p><p class="icon-twitter"></p><p class="icon-linkedin"></p><p class="icon-mail"></p></div>
				</div>
				<div class="ourteam_people">
					<img src="img/people.jpg" alt="" class="people_photo">
					<h4 class="name">Kate Upton</h4>
					<p class="position">Creative Director </p>
					<p class="description">Nemo enim ipsam voluptas sit aspernatur aut odit aut fugit, sed quia consequuntur magni dolores eos qui ratione voluptatem nesciunt.</p>
					<div class="people_sotialnetworks"><p class="icon-twitter"></p><p class="icon-linkedin"></p><p class="icon-mail"></p></div>
				</div>
				<div class="ourteam_people">
					<img src="img/people.jpg" alt="" class="people_photo">
					<h4 class="name">Olivia Wilde</h4>
					<p class="position">Lead Designer</p>
					<p class="description">Nemo enim ipsam voluptas sit aspernatur aut odit aut fugit, sed quia consequuntur magni dolores eos qui ratione voluptatem nesciunt.</p>
					<div class="people_sotialnetworks"><p class="icon-facebook"></p><p class="icon-twitter"></p><p class="icon-linkedin"></p><p class="icon-mail"></p></div>
				</div>
				<div class="ourteam_people">
					<img src="img/people.jpg" alt="" class="people_photo">
					<h4 class="name">Ashley Greene</h4>
					<p class="position">SEO / Developer </p>
					<p class="description">Nemo enim ipsam voluptas sit aspernatur aut odit aut fugit, sed quia consequuntur magni dolores eos qui ratione voluptatem nesciunt.</p>
					<div class="people_sotialnetworks"><p class="icon-twitter"></p><p class="icon-linkedin"></p><p class="icon-mail"></p></div>
				</div>
			</div>
		</div>
	</div>

	<div class="skills">
	<div class="container">
		<h2 class="skills_title">WE GOT SKILLS!</h2>
		<p class="skills_subtitle">Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod 
tempor incididunt ut labore et dolore magna aliqua.</p>
		<div class="skills__charts">
			
			<div class="skills__charts--web">
				<div class="charts__props" id="skills__chart_web"><div class="inner">90%</div> </div>
				<h5>web design</h5>
			</div>

			<div class="skills__charts--css">
				<div class="charts__props" id="skills__chart_css"><div class="inner">75%</div></div>
				<h5 class="">CSS</h5>
			</div>

			<div class="skills__charts--graphic">
				<div class="charts__props" id="skills__chart_graphic"><div class="inner">70%</div></div>
				<h5>graphic design</h5>
			</div>

			<div class="skills__charts--uiux">
				<div class="charts__props" id="skills__chart_uiux"><div class="inner">85%</div></div>
			<h5 >UI/UX</h5>
		</div>

		
	</div>
</div></div>


<div class="portfolio">
	<div class="container">
		<h2 class="portfolio_title">OUR PORTFOLIO</h2>
		<p class="portfolio_subtitle">Neque porro quisquam est, qui dolorem ipsum quia dolor sit amet
 consectetur, adipisci velit, sed quia non numquam</p>
		<div class="portfolio_buttons_row">
			<button class="portfolio_buttons_row__app selected">all</button>
			<button class="portfolio_buttons_row__app">web</button>
			<button class="portfolio_buttons_row__app">apps</button>
			<button class="portfolio_buttons_row__app">icons</button>
		</div>
		<div class="portfolio_apps_screenshots">
			<div class="portfolio_apps_screenshots__FirstRow">
			<div class="portfolio_apps_screenshot__block">
				<img src="img/mockup.png" alt="mockup">
				<p>Isometric Perspective Mock-Up</p>
			</div>
			<div class="portfolio_apps_screenshot__block">
				<img src="img/timezone.png" alt="timezone">
				<p>Time Zone App UI</p>
			</div>
		</div>


			<div class="portfolio_apps_screenshots__SecondRow">
			<div class="portfolio_apps_screenshot__block">
				<img src="img/media_players.png" alt="media players">
				<p>Viro Media Players UI</p>
			</div>
			<div class="portfolio_apps_screenshot__block">
				<img src="img/ui_kit.png" alt="Blog / Magazine Flat UI Kit">
				<p>Blog / Magazine Flat UI Kit</p>
			</div>
		</div>
		</div>
		<button class="portfolio_loadmore">LOAD MORE PROJECTS</button>
	</div>
</div>

<div class="reports">
	<div class="container">
		<h1 class="reports__title">WHAT POEPLE SAY ABOUT US</h1>
		<p class="reports__subtitle">Our clients love us!</p>
		<div class="reports__commentBlock">
			<div class="reports__first_row">
			<div class="comment_block">
				<img src="img/people.jpg" alt="">
				<div class="comment_body">
					<div class="comment__text">“Nullam dapibus blandit orci, viverra gravida dui lobortis eget. Maecenas fringilla urna eu nisl scelerisque.”</div>
					<div class="comment__name">Chanel Iman</div>
					<div class="comment__position">CEO of Pinterest</div>
				</div>
			</div>
			<div class="comment_block">
				<img src="img/people.jpg" alt="">
				<div class="comment_body">
					<div class="comment__text">“Nullam dapibus blandit orci, viverra gravida dui lobortis eget. Maecenas fringilla urna eu nisl scelerisque.”</div>
					<div class="comment__name">Chanel Iman</div>
					<div class="comment__position">CEO of Pinterest</div>
				</div>
			</div></div>

			<div class="reports__second_row">
			<div class="comment_block">
				<img src="img/people.jpg" alt="">
				<div class="comment_body">
					<div class="comment__text">“Nullam dapibus blandit orci, viverra gravida dui lobortis eget. Maecenas fringilla urna eu nisl scelerisque.”</div>
					<div class="comment__name">Chanel Iman</div>
					<div class="comment__position">CEO of Pinterest</div>
				</div>
			</div>
			<div class="comment_block">
				<img src="img/people.jpg" alt="">
				<div class="comment_body">
					<div class="comment__text">“Nullam dapibus blandit orci, viverra gravida dui lobortis eget. Maecenas fringilla urna eu nisl scelerisque.”</div>
					<div class="comment__name">Chanel Iman</div>
					<div class="comment__position">CEO of Pinterest</div>
				</div>
			</div>
		</div>
		</div>
	</div>
</div>


<div class="getintouch">
	<div class="container">
		<h1 class="getin__title">GET IN TOUCH</h1>
		<div class="getin__subtitle">1600 Pennsylvania Ave NW, Washington, DC 20500, United States of America. Tel: (202) 456-1111</div>
		<form action="">
			<div class="getin__Row">
				<input type="text" class="input__name" placeholder="Your Name *">
				<input type="text" class="input__email" placeholder="Your Email *">
			</div>

			<!--<input type="text" class="input__message" placeholder="Your Message *">-->
			<textarea name="" id="" cols="30" rows="10" class="input__message" placeholder="Your Message *"></textarea>
			<input class="submit" type="submit" value="SEND MESSAGE">
		</form>
	</div>
</div>

<footer>
	<div class="container">
		<div class="footer__socials">
			<p>Facebook</p>
			<p>Twitter</p>
			<p>LinkedIn</p>
			<p>Behance</p>
			<p>Dribble</p>
			<p>GitHub</p>
		</div>
	</div>
</footer>

<script src="node_modules/progressbar.js/dist/progressbar.min.js"></script>
<script >
	var bar = new ProgressBar.Circle(skills__chart_web, {
  color: '#30bae7',
  trailColor: '#eee',
  trailWidth: 1,
  duration: 2000,
  easing: 'bounce',
  strokeWidth: 8,
  from: {color: '#30bae7', a:0},
  to: {color: '#30bae7', a:1},


  // Set default step function for all animate calls
  step: function(state, circle) {
    circle.path.setAttribute('stroke', state.color);
    
  }
});

bar.animate(0.9); 

var bar = new ProgressBar.Circle(skills__chart_css, {
  color: '#d74680',
  trailColor: '#eee',
  trailWidth: 1,
  duration: 1400,
  easing: 'bounce',
  strokeWidth: 8,
  from: {color: '#d74680', a:0},
  to: {color: '#d74680', a:1},
  // Set default step function for all animate calls
  step: function(state, circle) {
    circle.path.setAttribute('stroke', state.color);
  }
});

bar.animate(.75); 

var bar = new ProgressBar.Circle(skills__chart_graphic, {
  color: '#15c7a8',
  trailColor: '#eee',
  trailWidth: 1,
  duration: 1400,
  easing: 'bounce',
  strokeWidth: 8,
  from: {color: '#15c7a8', a:0},
  to: {color: '#15c7a8', a:1},
  // Set default step function for all animate calls
  step: function(state, circle) {
    circle.path.setAttribute('stroke', state.color);
  }
});

bar.animate(.7); 

var bar = new ProgressBar.Circle(skills__chart_uiux	, {
  color: '#eb7d4b',
  trailColor: '#eee',
  trailWidth: 1,
  duration: 1400,
  easing: 'bounce',
  strokeWidth: 8,
  from: {color: '#eb7d4b', a:0},
  to: {color: '#eb7d4b', a:1},
  // Set default step function for all animate calls
  step: function(state, circle) {
    circle.path.setAttribute('stroke', state.color);
  }
});

bar.animate(.85); 
</script>

</body>
</html>