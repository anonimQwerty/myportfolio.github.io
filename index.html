<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<meta name="viewport" content="width=device-width, initial-scale=1.0">
	<title>My portfolio</title>
</head>
<body>
	<p>Крч мне пока делать главный макет влом, поэтому вот ссылки на мои сайты:</p>
	<ol>
		<li><a href="cuda/index.html">Cuda</a></li>
		<li><a href="golden/index.html">Golden</a></li>
	</ol>
</body>
</html>
