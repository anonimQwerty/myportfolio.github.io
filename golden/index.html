<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<meta name="viewport" content="width=device-width, initial-scale=1.0">
	<link rel="stylesheet" href="css/main.css">
	<title>Document</title>
</head>
<body>
	<div class="first_block">
		<div class="pannel"></div>
		<div class="container">

			<header>
				<img src="img/logo.png" alt="">
				<i class="burger fas fa-bars"></i>

				<nav>
					<a href="#">home</a>
					<a href="#">servises</a>
					<a href="#">portfolio</a>
					<a href="#">about</a>
					<a href="#">contact</a>
				</nav>
			</header>

			<h2>Welcome To Our Studio!</h2>
			<h1>It’s nice to Meet you</h1>
			<button>tell me more</button>
		</div>
	</div>

	<div class="container">
		<div class="servises">
			<h2 class="servises__title">servises</h2>
			<p class="servises__subtitle">Proin iaculis purus consequat sem cure.</p>
			<div class="servises__servises">
				<div class="servise__card">
					<img src="img/e_commerce.png" alt="" class="servise__card--image">
					<div class="servise__card--title">E-Commerce</div>
					<div class="servise__card--subtitle">Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt.</div>
				</div>
				<div class="servise__card">
					<img src="img/responsible_web.png" alt="" class="servise__card--image">
					<div class="servise__card--title">Responsible Web</div>
					<div class="servise__card--subtitle">Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt.</div>
				</div>
				<div class="servise__card">
					<img src="img/web_security.png" alt="" class="servise__card--image">
					<div class="servise__card--title">Web Security</div>
					<div class="servise__card--subtitle">Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt</div>
				</div>
			</div>
		</div>

		<div class="portfolio">
			<h2 class="portfolio__title">portfolio</h2>
			<p class="portfolio__subritle">Proin iaculis purus consequat sem cure.</p>
			<div class="portfolio__img">
				<img src="img/prtfolio_ebony.png" alt="">
				<img src="img/portfolio_2.png" alt="">
				<img src="img/portfolio_3.png" alt="">
			</div>
		</div>

		<div class="about">
			<div class="about__title">About Us</div>
			<div class="aboout__subtitle">Proin iaculis purus consequat sem cure.</div>
			<div class="about__cars">
				

					<div class="card__textblock posleft">
						<h5 class="date">July 2010</h5>
						 <h5 class="whatHappen">Our Humble Beginnings</h5>
						<p>Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt. Proin iaculis purus consequat sem cure digni.</p>
					</div>
					<img class="about__photo" src="img/about1.png"></img>
					


					<div class="card__textblock posright" id="sec_text">
					
						<h5 class="date">January 2011</h5>
						<h5 class="whatHappen">Facing Startup Battles</h5>
						<p>Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt. Proin iaculis purus consequat sem cure digni.</p>
					</div>
					<img class="about__photo" id="sec_photo" src="img/about2.png"></img>
					
					
				
					<div class="card__textblock posleft" id="thrd_rext">
						<h5 class="date">December 2012</h5>
<h5 class="whatHappen">Enter The Dark Days</h5>
						<p>Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt. Proin iaculis purus consequat sem cure digni.</p>
					</div>
					<img class="about__photo" src="img/about3.png"></img>
					
				

					
					<div class="card__textblock posright" id="forth_text">
						<h5 class="date">february 2014</h5>
<h5 class="whatHappen">Our Triumph</h5>
						<p>Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt. Proin iaculis purus consequat sem cure digni.</p>

					
					
				</div>
				<img class="about__photo" src="img/about4.png"></img>

				
			</div>

		</div>

		<footer>
			<p>© Copyright 2014 FreebiesXpress.com</p>
		</footer>




	</div>

<script src="https://kit.fontawesome.com/a6dbdcd920.js" crossorigin="anonymous"></script>
<script src="https://code.jquery.com/jquery-3.6.0.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script>
</body>
</html>